pom.xml

```diff
@@ -1,313 +1,313 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.laserfiche</groupId>
-  <artifactId>lf-repository-api-client</artifactId>
-  <packaging>jar</packaging>
-  <name>Laserfiche Repository API Client</name>
-  <version>1.0.0</version>
-  <url>https://github.com/Laserfiche/lf-repository-api-client-java</url>
-  <description>Java implementation of various foundational APIs for Laserfiche, including Repository APIs such as
-    Entry API flows for secure and easy access to Laserfiche Repository Entries.</description>
-  <scm>
-    <connection>scm:git:git://github.com/Laserfiche/lf-repository-api-client-java.git</connection>
-    <developerConnection>scm:git:ssh://github.com:Laserfiche/lf-repository-api-client-java.git</developerConnection>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.laserfiche</groupId>
+    <artifactId>lf-repository-api-client</artifactId>
+    <packaging>jar</packaging>
+    <name>Laserfiche Repository API Client</name>
+    <version>1.0.0</version>
     <url>https://github.com/Laserfiche/lf-repository-api-client-java</url>
-  </scm>
-  <prerequisites>
-    <maven>2.2.0</maven>
-  </prerequisites>
+    <description>Java implementation of various foundational APIs for Laserfiche, including Repository APIs such as
+        Entry API flows for secure and easy access to Laserfiche Repository Entries.
+    </description>
+    <scm>
+        <connection>scm:git:git://github.com/Laserfiche/lf-repository-api-client-java.git</connection>
+        <developerConnection>scm:git:ssh://github.com:Laserfiche/lf-repository-api-client-java.git</developerConnection>
+        <url>https://github.com/Laserfiche/lf-repository-api-client-java</url>
+    </scm>
+    <prerequisites>
+        <maven>2.2.0</maven>
+    </prerequisites>
 
-  <licenses>
-    <license>
-      <name>MIT</name>
-      <url>https://opensource.org/licenses/MIT</url>
-      <distribution>repo</distribution>
-    </license>
-  </licenses>
+    <licenses>
```

Comment:
Again, do we want to change the indentation?

Reply:
I replaced the original file.